doc/topics/the-event-loop-timers-and-nexttick.md

```diff
@@ -218,10 +218,10 @@ ways depending on when they are called.
 * `setTimeout()` schedules a script to be run
 after a minimum threshold in ms has elapsed.
 
-The order in which they are execute varies depending on the context in
-which they are called.  If both are called in the main module then you
-are bound to how fast your process go, which is impacted by other
-programs running on your machine.
+The order in which they are executed varies depending on the context in
+which they are called.  If both are called in the main module, then you
+are bound by how fast your process goes (which is impacted by other
+programs running on your machine).
```

Comment:
Would prefer to rework this a bit to avoid the use of `you` and `your`. Perhaps...

```
The order in which the timers are executed will vary depending on the context
in which they are called. If both are called from within the main module, then timing
will be bound by the performance of the process (which can be impacted by other
applications running on the machine).
```

Or something along those lines.